index.html

```diff
@@ -216,5 +216,82 @@ <h2 class="aboutTitle">Skills</h2>
         </li>
       </ul>
     </section>
+    <section class="contact">
+      <div class="contactHeader">
+        <h2 class="contactTitle">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+
+      <form
+        class="contactForm"
+        action="https://formspree.io/f/mknarzzq"
+        method="post"
+      >
+        <input
+          id="fullName"
+          name="fullName"
+          type="text"
+          placeholder="Full Name"
+          required
+        />
```

Comment:
- [ ] Kindly, update the name's `input` tag to take in a maximum of 30 characters, by making use of the `maxlength` attribute of the `input`s/`textarea` tags, as suggested in the requirement. 😀